common/match/pool.ts

```diff
@@ -540,7 +541,22 @@ export async function sendConfirmationRequests(pool: MatchPool) {
     }
 }
 
+// Add a pending pupil screening to every pupil that needs one
+async function addPupilScreenings() {
+    // find all pupils that do not have a valid screening
+    const pupils = await prisma.pupil.findMany({ where: { pupil_screening: { none: { invalidated: false } } } });
```

Comment:
We don't want to request a pupil screening from everyone?

Reply:
I would propose to 
- add two new toggles analogous to "confirmation-pending", "confirmation-unknown", i.e. "screening-pending" and "screening-unknown"
- Adapt `getPupilsToConfirmInterest` to `getPupilsToContactNext(toggle: "confirmation-unknown" | "screening-unknown")`
- Add a toggle to the Match Pool, analogous to `pool.confirmInterest`, i.e. `pool.needsScreening`. Then we can toggle of confirmInterest, manually request pupil screenings, and once the process switch is done we can toggle on needsScreening and the automatic job takes over